Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/ImageClassifier/ImageClassifier.xaml

```diff
@@ -75,7 +75,7 @@
                                 <ComboBox x:Name="DeviceComboBox" SelectedIndex="0" Background="LightGray" PlaceholderText="Device" Height="50" Width="97"
                                           SelectionChanged="DeviceComboBox_SelectionChanged">
                                     <TextBlock Text="CPU" FontSize="18" Typography.Capitals="AllSmallCaps" Typography.StylisticSet4="True"/>
-                                    <TextBlock Text="DML" FontSize="18" Typography.Capitals="AllSmallCaps" Typography.StylisticSet4="True"/>

```

Comment:
Do we want to call our DML here (as opposed to GPU) for upsell?

Reply:
I wanted it to be consistent. I'm fine with either "GPU" or "DirectML" (we shouldn't use "DML"). You prefer DirectML?